<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSlideText Example</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSlideText" name="_qslidetext" native="true">
     <property name="text" stdset="0">
      <string>Still Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlideText" name="_qslidetext_2" native="true">
     <property name="text" stdset="0">
      <string>Slide Mode Normal</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlideText" name="_qslidetext_3" native="true">
     <property name="text" stdset="0">
      <string>Intermittent Mode Slow</string>
     </property>
     <property name="textColor" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlideText" name="_qslidetext_4" native="true">
     <property name="text" stdset="0">
      <string>Still Algin Right</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlideText" name="_qslidetext_5" native="true">
     <property name="text" stdset="0">
      <string>Intermittent Mode Algin Left</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlideText" name="_qslidetext_6" native="true">
     <property name="text" stdset="0">
      <string>Very Long Text Message in Slide Mode and Very Fast.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSlideText</class>
   <extends>QWidget</extends>
   <header location="global">qslidetext.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
